### PR TITLE
docs: Improve the WebUI page

### PR DESCRIPTION
**Pull request by tara-det-ai**
Make the WebUI page more user friendly and improve algolia and google search crawler results by adding headings and links to more information and ways to work with the WebUI.

### CODE REVIEW

File: docs/interfaces/webui-if.rst [hunk 1]
```diff
@@ -4,12 +4,93 @@
...
+you to the login page. Once you log in, you will be redirected to the initial URL you entered.
+
+To end the Determined session and log out of the WebUI, click **Sign Out**.
+
+************************
+ Creating an Experiment
+************************
+
+If you have an existing experiment or trial, you can use the WebUI to create an experiment. To find
+out more, visit :ref:`experiments`.
```
Comment (wes-turner) [inline docs/interfaces/webui-if.rst:27]: nit: This wording a little bit makes me think that clicking through will tell me more about using the web UI to create experiments. Instead, it's about creating experiments _in general_.

For me and my expectations I'd prefer seeing this as like
> To learn more about creating experiments, visit :ref:`experiments`.

But that could well take it away from a style that you're going for. Whatever you think is best, you got my support.

File: docs/interfaces/webui-if.rst [hunk 2]
```diff
@@ -4,12 +4,93 @@
...
+**********************************************************
+
+The WebUI lets you organize your experiments into projects and workspaces. To get started, visit
+:ref:`workspaces`.
+
+*******************************************
+ Configuring the Behavior of an Experiment
+*******************************************
+
+Many experiment configuration settings can be performed via the WebUI. To find out more, visit
```
Comment (wes-turner) [inline docs/interfaces/webui-if.rst:40]: Same comment as above applies here and below (and to a lesser extent in "workspaces and projects"). Whichever route you like.

File: docs/interfaces/webui-if.rst [hunk 3]
```diff
@@ -4,12 +4,93 @@
...
+:ref:`experiment-config-reference`.
+
+***************************************
+ Evaluating the Performance of a Model
+***************************************
+
+You can access training and validation performance information via the WebUI. To see model
+evaluation in action, follow the steps described in the :ref:`pytorch-mnist-tutorial`.
```
Comment (wes-turner) [inline docs/interfaces/webui-if.rst:48]: This is a nice addition.

File: docs/interfaces/webui-if.rst [hunk 4]
```diff
@@ -4,12 +4,93 @@
...
+***********************
+
+To find what kind of anonymous information the WebUI collects, visit
+:ref:`common-configuration-options`.
+
+************************
+ Managing the Job Queue
+************************
+
+To find out how to view and modify the Job Queue in the WebUI, visit :ref:`experiments`.
```
Comment (wes-turner) [inline docs/interfaces/webui-if.rst:84]: Could this connect to the `#view-the-job-queue` anchor instead?

File: docs/interfaces/webui-if.rst [hunk 5]
```diff
@@ -4,12 +4,93 @@
...
+ Starting a Jupyter Notebook
+*****************************
+
+You can start :ref:`notebooks` from the WebUI.
+
+***********************
+ Launching TensorBoard
+***********************
+
+One way to launch TensorBoard is from the WebUI. To find out more, visit :ref:`tensorboards`.
```
Comment (wes-turner) [inline docs/interfaces/webui-if.rst:96]: nit: I know you're at this point probably tired of writing "you can X from the WebUI!" but I think that formulation is slightly more "user manual" style. Totally your choice, just calling this out for you to take another look at.

**Review by wes-turner — APPROVED** [review]

File: docs/interfaces/webui-if.rst (hunk 1 above)
Comment (tara-det-ai) [reply docs/interfaces/webui-if.rst:27]: good point, i've updated it to let the user know they'll need to use the CLI to create a "new" experiment

File: docs/interfaces/webui-if.rst (hunk 2 above)
Comment (tara-det-ai) [reply docs/interfaces/webui-if.rst:40]: i agree; changes made

File: docs/interfaces/webui-if.rst (hunk 4 above)
Comment (tara-det-ai) [reply docs/interfaces/webui-if.rst:84]: yes, good catch

File: docs/interfaces/webui-if.rst (hunk 5 above)
Comment (tara-det-ai) [reply docs/interfaces/webui-if.rst:96]: yes that's a better way to say it, resolved

**Pull request merged**